Compile broken

$ java -version
java version "1.7.0_79"
Java(TM) SE Runtime Environment (build 1.7.0_79-b15)
Java HotSpot(TM) 64-Bit Server VM (build 24.79-b02, mixed mode)

A simple mvn compile at the root on a fresh checkout fails:

ERROR] The build could not read 1 project -> [Help 1]
[ERROR]   
[ERROR]   The project com.google.oauth-client:dailymotion-simple-cmdline-sample:[unknown-version] (/Users/elharo/google-oauth-java-client/samples/dailymotion-cmdline-sample/pom.xml) has 1 error
[ERROR]     Non-resolvable parent POM for com.google.oauth-client:dailymotion-simple-cmdline-sample:[unknown-version]: Could not find artifact com.google.oauth-client:google-oauth-client-parent:pom:1.22.0-SNAPSHOT and 'parent.relativePath' points at wrong local POM @ line 4, column 11 -> [Help 2]
[ERROR] 
[ERROR] To see the full stack trace of the errors, re-run Maven with the -e switch.
[ERROR] Re-run Maven using the -X switch to enable full debug logging.
[ERROR] 
[ERROR] For more information about the errors and possible solutions, please read the following articles:
[ERROR] [Help 1] http://cwiki.apache.org/confluence/display/MAVEN/ProjectBuildingException
[ERROR] [Help 2] http://cwiki.apache.org/confluence/display/MAVEN/UnresolvableModelException

--- a/samples/dailymotion-cmdline-sample/pom.xml
+++ b/samples/dailymotion-cmdline-sample/pom.xml
@@ -4,7 +4,7 @@
   <parent>
     <groupId>com.google.oauth-client</groupId>
     <artifactId>google-oauth-client-parent</artifactId>
-    <version>1.22.0-SNAPSHOT</version>
+    <version>1.23.0-SNAPSHOT</version>
     <relativePath>../../pom.xml</relativePath>
   </parent>
   <artifactId>dailymotion-simple-cmdline-sample</artifactId>

